### DISCUSSION TITLE
Lags in large files. 31k loc in single file.

### DISCUSSION THREAD

**Discussion by MinSomai** [ecosse3/nvim#117] [Ideas]
Lags before opening and can't even type.
The ram usage is normal too. CPU doesn't even cross 50%.

I can provide diagnostics if you want to debug. I can't seem to figure out the reason.

**Comment by ecosse3** [comment]
Did you try to disable `treesitter` for this specific filetype or completely to test it? It usually causes performance problems in such big files.

**Reply by MinSomai** [reply]
Tried commenting out all the treesitter related plugins in `plugins.lua`, the issue persists.
Currently using LunarVim unfortunately.

I once had similar issue with larger files because of the git blame plugin in vanilla `init.vim` file. Using another similar plugin solved the issue.
